README.md

```diff
@@ -17,8 +34,17 @@ To deploy, create a playbook which looks like this:
             control: "{{ inventory_hostname in groups['cluster_control'] }}"
             batch: "{{ inventory_hostname in groups['cluster_batch'] }}"
             runtime: true
+          openhpc_slurm_service_enabled: true
+          openhpc_slurm_service: slurmd
+          openhpc_slurm_control_host:
```

Comment:
Maybe set this as `openhpc_slurm_control_host: "{{ groups['cluster_login'] | first }}"`

Reply:
If we were referring to groups we shouldn't make assumptions on their actual names, ideally.

Reply:
My mistake, didn't notice this was the README... @brtknr is right